### PR TITLE
Move sql models into appropriate folder

### PR DESCRIPTION
**Pull request by Ma-wa-re**
Moved all models into a models folder and added a naming convention for model names

### CODE REVIEW

File: futaba/sql/models/settings_model.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# sql/settings.py
+# sql/models/settings_model.py
```
Comment (ammongit) [inline futaba/sql/models/settings_model.py]: We were considering two routes, either `models/settings` or `settings_model`. I don't think we need both.

File: futaba/cogs/settings/settings.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# cogs/settings.py
+# cogs/settings/settings.py
```
Comment (ammongit) [inline futaba/cogs/settings/settings.py]: We should have standard naming for cogs, if those files are needed. This should be `commands.py` or something imo.

File: futaba/sql/models/settings_model.py (hunk 1 above)
Comment (Ma-wa-re) [reply futaba/sql/models/settings_model.py]: Ok, I'll change it to /models/<file>.py as that looks better

File: futaba/cogs/settings/settings.py (hunk 2 above)
Comment (Ma-wa-re) [reply futaba/cogs/settings/settings.py]: I agree, having the commands in commands make sense. However I think there should be a core.py that holds the class for the cog and any other file can be named accordingly

Comment (ammongit) [reply futaba/cogs/settings/settings.py]: Sounds good!

File: futaba/cogs/_core.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#
+# cogs/<Cog Folder Name>/core.py
```
Comment (ammongit) [inline futaba/cogs/_core.py]: Should be lowercase?

**Pull request merged**